### DISCUSSION TITLE
Models (how to choose the right ones)

### DISCUSSION THREAD

**Discussion by ACEXAN** [intel/openvino-plugins-ai-audacity#203] [Q&A] [answered]
First of all, thank you, it seems to be a great tool! 
This probably gonna sound stupid but, during the installation, how does one know which models to install and what each one does (better)? The list is quite long. For example, there's quite a list of models for Whisper (long 1, long 2, long 3, short etc...) does each one do something specific? The smaller the better? The bigger the better? Or should I just install all of them?
Thanks in advance!
P.S. I looked for similar discussions, I swear!..

**Comment by RyanMetcalfeInt8** [comment, marked as answer]
Hi @ACEXAN,

It's not a stupid question -- we definitely do need to provide some more information at the 'select models' phase. As you said, unless we informed about the pros/cons of each model, it's hard to know what to pick.

For whisper -- as the model size increases, the transcription (and especially translation) accuracy will increase -- but the processing time is much longer. I wouldn't necessarily install all of them -- perhaps base, small, medium, and large-v3, but even that might be overkill. It also depends on what you want to do with it. If you want to use it to produce usable subtitles, for example, I would probably use a larger model like medium or large. On the other hand, if you want to produce a label track only to use as a 'guide' for your project -- running 'base' is probably just fine.

**Reply by ACEXAN** [reply]
I see. So since I mostly plan on transcribing my audio notes, I should fly somewhere between base and medium. And when the time comes for some serious (subtitles, etc...) "heavy lifting" I'll add larger models.
Thank you!

**Comment by The3IC** [comment]
If I may chime in here. Done some testing with different models and quite frankly, (at least for English), the large-v3 is the best. Yes, it takes a bit longer (about 1 minute of processing time on my machine for minute of audio, compared to eg large-v2 which seems to be about half of that) but if you want to create quality subtitles for eg Youtube, the editing work to fix any remaining errors is just the least in that. So for anything going public, my personal opinion is that large-v3 is the way to go!
